TestApp/ViewController.swift

```diff
@@ -21,7 +21,19 @@ class ViewController: UIViewController {
         self.displayText(text: "danhger-swift3546", times: 5)
         print("code merged & jenkins file changed")
         print("code mer++++")
-        print("ghjghgh")
+        print("ghjghghhdhdh")
+        
+         print("code merged & jenkins file changed")
+        print("code mer++++")
+        
+        
+         print("code merged & jenkins file changed")
+        print("code mer++++")
+        
```

Comment:
<!--
  0 failure: 
  1 warning:  Files should have...
  
  
  DangerID: danger-id-Danger;
  File: TestApp/ViewController.swift;
  Line: 32;
-->

- :warning: Files should have a single trailing newline. (`trailing_newline`)